lib/LaTeXML/Package/graphics.sty.ltxml

```diff
@@ -221,8 +221,11 @@ DefConstructor('\graphicspath DirectoryList', sub {
   properties => sub {
     my ($stomach, $paths) = @_;
     my @paths = ();
+    my $root  = $STATE->lookupValue('SOURCEDIRECTORY') || '';
     foreach my $dir ($paths->getValues) {
-      my $path = pathname_absolute(pathname_canonical(ToString($dir)));
+      $dir = pathname_canonical(ToString($dir));
+      $dir = pathname_concat($root, $dir) if $root && !pathname_is_absolute($dir);
+      my $path = pathname_absolute($dir);
```

Comment:
Is this better (or different) than simply passing `$root` as the base directory, as in: ```$path=pathname_absolute(pathname_canonical(ToString($dir)), $root);```
since that's what that argument is intended for?

Reply:
Oh, my oversight that this was supported so easily - upgraded, thanks!